### ISSUE TITLE
Clarify interaction between Prezi and Image service

### ISSUE THREAD

**Issue by azaroth42** [IIIF/api#1798]
Clarify that:

* The service URI should be the base URI (per `service` description) not the info.json URI
* That the service has to be associated with an image, not as the body of the annotation directly.

From #1769 

```
{
  "@context": "http://iiif.io/api/presentation/3/context.json",
  "id": "https://example.org/iiif/book1/annotation/p0001-image",
  "type": "Annotation",
  "motivation": "painting",
  "body": {
    "id": "https://example.org/images/book1-page1/full/full/0/default.jpg",
    "type": "Image",
    "label": {"en": ["Page 1"], "es": ["Página 1"]},
    "format": "image/jpeg",
    "service": [
      {
        "id": "https://example.org/images/book1-page1/info.json",
        "type": "ImageService3",
        "profile": "level2"
      }
    ],
    "height": 2000,
    "width": 1500
  },
  "target": "https://example.org/iiif/book1/canvas/p1"
}
```

**Comment by azaroth42** [comment]
The spec says in section 5.7:

> If the content resource is an Image, and a IIIF Image service is available for it, then the URI _MAY_ be a complete URI to any particular representation made available, such as `https://example.org/image1/full/1000,/0/default.jpg`, but _MUST NOT_ be just the URI of the IIIF Image service. Its `type` value _MUST_ be the string `Image`.

I think this is pretty explicit that the example from @ahankinson is invalid.

**Comment by tomcrane** [comment]
I was going to add something along those lines but it's already there. 

The text in 5.7 is where the Presentation API meets the Image API; it's worth spending a lot of effort on it.

**Comment by azaroth42** [comment]
Close, already fixed?

**Comment by tomcrane** [comment]
I'll experiment with the wording a bit.

**Comment by azaroth42** [comment]
Closed by #1808
